src/ert/config/ert_config.py

```diff
@@ -760,6 +763,9 @@ class ErtConfigWithPlugins(ErtConfig):
             ENV_PR_FM_STEP: ClassVar[dict[str, dict[str, Any]]] = env_pr_fm_step
             ACTIVATE_SCRIPT = pm.activate_script()
 
+        from datetime import datetime  # noqa
```

Comment:
This is already imported at the top

Reply:
I am not sure why, but without this I get:
```
NameError: name 'datetime' is not defined
```

Reply:
Probably due to some circular dependency somewhere

Reply:
hm, I'd then suggest to fix the circular imports first or?

Reply:
Think it is related to this:  https://github.com/equinor/ert/issues/9587, so long standing